packages/vscode-extension-tester/server/webSocketServer.js

```diff
@@ -29,30 +29,43 @@ export function initServer(port = 8080) {
   return wss;
 }
 
+export function resetAppWebsocket() {
+  appWebsocket = null;
+}
+
 export function getAppWebsocket() {
   return appWebsocket;
 }
 
-export function waitForMessage(id, timeoutMs = 5000) {
+export function waitForMessage(id, timeoutMs = 10000) {
   return new Promise((resolve, reject) => {
     const appWebsocket = getAppWebsocket();
-    if (!appWebsocket) {
-      reject(new Error("No websocket connection"));
+
+    if (!appWebsocket || appWebsocket.readyState !== 1) {
+      reject(
+        new Error(
+          `Websocket not connected or not ready. State: ${
+            appWebsocket ? appWebsocket.readyState : "null"
```

Comment:
NIT:
Isn't this the same as

```suggestion
            appWebsocket?.readyState
```